cellprofiler/modules/exporttocellh5.py

```diff
@@ -366,35 +392,84 @@ def keep(column):
         # it's clearer how it's organized. I'm expecting that you would
         # organize it differently when actually storing.
         #
-        for column in columns:
-            object_name, feature_name = column[:2]
-            if not m.has_feature(object_name, feature_name):
-                continue
-            if object_name == cpmeas.EXPERIMENT:
-                continue
-            if object_name == cpmeas.IMAGE:
-                value = m[object_name, feature_name]
-                #
-                # TODO: this is an image-wide feature. If there's no concept
-                #       of that, I would create an all-1's segmentation as the
-                #       objects for these measurements
-                #
-                # store a measurement with feature name "feature_name"
-                # and value, "value"
-            else:
+        
+        ### 0) extract object information (i.e. object_label_no) 
+        ### 1) extract all single cell features and write it as feature matrix (for e.g. classification)
+        ### 2) extract Center
+        ### 3) create artifical Bounding box... usefull for displaying it in fiji lateron
+        ### 4) Don't see the point of features extracted on "Image" the only real and useful feature there is "Count" which can be deduced from single cell information
```

Comment:
Q/A paramteres are indeed usefull, I think of how to add this to cellh5. We haven't had a good use case for this, so far. Definitely, I want to avoid storing "all ones" pixel images to it